### PR TITLE
fix: file not found error in GH Action workflow

### PR DESCRIPTION
**Pull request by covalentbond**
Signed-off-by: Arijit kunduarijit55@gmail.com

The run seems to fail at `Run ics-to-yml.py`:

image

The current change should possibly sove this.

@jmertic for review, though it might again fail at next step: `Check diff` saying "Not a git repository" (Not sure about this)

### CODE REVIEW

**Comment by jmertic** [comment]
Hey @covalentbond - two things...

1) This script is available in the web portion at https://www.zowe.org/assets/script/ics-to-yml.py - you might want to stick it under `.github` somewhere instead to not have it exposed there.
2) You can set the directory to run the script out of in the workflow definition. That might be a better way to solve it.

**Comment by covalentbond** [comment]
Hey @jmertic , can you review it once. I changed the directory of the script under `_data` itself. This doesn't seem to give the `FileNotFoundError` when tested in another repository!

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Events are not supposed to be added/modified manually, since they directly get s
 
 The events are dynamically fetched from an [ICS feed](https://www.openmainframeproject.org/events/category/zowe?ical=1&tribe_display=photo) and parsed into the [upcoming_events.yml](_data/upcoming_events.yml).
 
-If needed, the [upcoming_events.yml](_data/upcoming_events.yml) can be manually updated by running `python assets/script/ics-to-yml.py` locally from root directory.
+If needed, the [upcoming_events.yml](_data/upcoming_events.yml) can be manually updated by running `cd _data` and `python _data/_ics-to-yml.py` locally from root directory.
```
Comment (jmertic) [inline README.md:42]: If you are in the `_data` directory, you don't need to prefix the python script you are running with that directory.

**Comment by jmertic** [comment]
Should be all good @covalentbond - one comment for the documentation.

File: README.md (hunk 1 above)
Comment (covalentbond) [reply README.md]: Thanks just corrected it, might have done accidentally!

**Review by jmertic — APPROVED** [review]

**Pull request merged**